### DISCUSSION TITLE
Schalter mit mehreren Ausprägungen

### DISCUSSION THREAD

**Discussion by Paddelinho** [thkl/hap-homematic#466] [Ideas]
Hallo,
ich nutze Rollläden mit einer speziellen Jalousiefunktion. Die sind so eingestellt, dass sie drei Zustände erlauben: offen, geschlossen, Jalousiemodus. Toll wäre es, wenn es einen Schalter geben würde, bei dem man zwischen mehr als zwei Zuständen auswählen könnte, ähnlich wie beim Alarmschalter nur mit freier Konfiguration der Bezeichnungen. Lässt sich so etwas realisieren? Der Anwendungsbereich für solche Schalter geht über mein spezielles Problem sicher weit hinaus.
Grüße

PS: Aktuell habe ich drei Switches implementiert, einen für jeden Zustand. Die Programmierung der Abhängigkeiten zwischen ihnen ist aber komplex und fehleranfällig.

**Comment by Mcarbuhn** [comment]
Moin, 
an selbigen Schalter wäre ich auch interessiert um das Schalten von Wochenprogrammen zu ermöglichen. Ich habe unter Raspberrymatic Programme erstellt mit denen ich die Wochenprogramme über Virtuelle Schalter (3 Stück, kurzer Tastendruck) aktivieren kann. Nun habe ich die drei Schalter als Gerät implementiert um diese dann über die HomeKit App zu schalten. Usabilty technisch wäre es jetzt natürlich super komfortabel und übersichtlicher wenn man einen Schalter wie um Alarm hätte um die unterschiedlichen Programme zuschalten und gleichzeitig zu visualisieren. 

Eine Rückmeldung wäre super. Vielen Dank. 

Beste Grüße! 
Marv
